testssl/README.md

```diff
@@ -0,0 +1,13 @@
+# testssl.sh
+
+In order to test BCP-003-01, the 'testssl.sh' script (v3.0 rc4) must be placed in this directory with 'execute' permission enabled. This tool is known to work on Linux and Mac OS, but will not work on Windows.
+
```

Comment:
Will attempt to try on Windows 10, as per https://github.com/drwetter/testssl.sh/issues/337, unless you've already done so?

Reply:
I haven't as yet, so please feel free to try.